less/index.less

```diff
@@ -53,7 +53,208 @@ table {
     box-sizing: border-box;
 }
 
+
+
 html, body {
-    height: 100%;
+	height: 100%;
+}
+
+// minis
+.red-border {
+	border: solid red 1px;
+}
+
+.max-width {
+	max-width: 100%;
+}
+
+.display-flex {
+	display: flex;
+}
+
+// border colors
+@border-color-dark: black;
+@border-color-light: #dddddd;
+
+//font-size stack
+@text-small: 1em;
+@text-reg: 1.125em;
+@text-med: 1.25em;
+@text-large: 1.5em;
+@text-jumbo: 2em;
+
+
+// CSS Start
+
+
+// Global Element Selectors
+
+h1 {
+	color: #252122;
+	font-weight: 600;
+	font-size: @text-large;
+	letter-spacing: -0.035em;
+	line-height: 1;
+	margin: 0.5em 0;
 }
 
+h2, h3, h4, h5, h6 {
+	font-size: @text-med;
+	margin: 0.5em 0;
+}
+
+button {
+	background-color: #4CAF50; /* Green */
+	border: none;
+	border-radius: 0.2em;
+	color: white;
+	padding: 15px 32px;
+	text-align: center;
+	text-decoration: none;
+	display: inline-block;
+	font-size: @text-small;
+	margin: 12px 2px;
+	cursor: pointer;
+}
+
+p {
+	margin-bottom: 2em;
+	line-height: 1.4em;
+}
+
+img {
+	.max-width;
+    height: auto;
+}
+// Nested Container Selectors
+
+.container {
+	.max-width;
+	width: 70em;
+	margin: 0 auto;
+	// .red-border;
+
+		.nav {
+			margin: 0.5em 0 0 0;
+			
+
+			ul {
+				margin: 0;
+			}
+
+			li {
+				display: inline-block;
+				margin: 0 0.35em 0 0.35em;
```

Comment:
with 4 values they correspond to Top Right Bottom Left
but you could also use two values to alter TB RL;
so this line could also be written as
margin: 0 0.35em;